docs/source/learning_algorithm.rst

```diff
@@ -43,13 +43,14 @@ The following table shows the options that can be adjusted and gives a short exp
   batch_size                              The batch size of experience considered from the buffer for an update.
   gamma                                   The discount factor, with which future expected rewards are considered in the decision-making.
   device                                  The device to use.
-  noise_sigma                             The standard deviation of the distribution used to draw the noise, which is added to the actions and forces exploration.  noise_scale
-  noise_dt                                Determines how quickly the noise weakens over time.
+  noise_sigma                             The standard deviation of the distribution used to draw the noise, which is added to the actions and forces exploration.
+  noise_dt                                Determines how quickly the noise weakens over time / used for noise scheduling.
   noise_scale                             The scale of the noise, which is multiplied by the noise drawn from the distribution.
   early_stopping_steps                    The number of steps considered for early stopping. If the moving average reward does not improve over this number of steps, the learning is stopped.
   early_stopping_threshold                The value by which the average reward needs to improve to avoid early stopping.
  ======================================== ==========================================================================================================
 
+Options for (linear) scheduling of learning rate and action noise are available, but not settable in the config file.
```

Comment:
This is not true anymore?
So the two parameters should be added to the table above right?

Reply:
Yeah, you're right. Missed that - thanks! I'm adding them now.